### PR TITLE
Add community guidelines for contributions

### PR DESCRIPTION
**Pull request by SnehPandya18**
What does this PR add / remove or fix?
* Create Code of Conduct, Issue Template & Pull Request Template
* Update existing README with proper target links to the guides

Does this PR match the guidelines documented in the Contributing Guide?
* Yes

### CODE REVIEW

**Comment by manan-jadhav** [comment]
Please remove old PR template from contributing guide.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -245,12 +245,6 @@ Here, `add-post-on-devfest` is the name of the branch you created in the previou
 
 Your changes are now pushed to GitHub and you can see it by going to the repository page!
 
-### Create a Pull Request
```
Comment (manan-jadhav) [inline CONTRIBUTING.md]: This section needs to stay

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -318,21 +312,3 @@ As you can see, the assets are put in a directory with the post name, and that d
 It is _MUST_ to use either JPG or PNG images, and the filesize must be less than 250 KB in all cases. 
 
 Directly linking to external images is also _NOT_ allowed.
-
-## Writing Pull Requests
```
Comment (manan-jadhav) [inline CONTRIBUTING.md]: Add description that one should use the template provided, and link to the template. Keep this section also as it is being referenced from table of contents

**Review by manan-jadhav — CHANGES_REQUESTED** [review]

**Comment by manan-jadhav** [comment]
@SnehPandya18 Please provide corrections so we can push this.

**Comment by SnehPandya18** [comment]
@CurosMJ the requested changes have been incorporated. Kindly have a look.

**Review by manan-jadhav — APPROVED** [review]

**Pull request merged**